<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LED_current_limit</class>
 <widget class="QWidget" name="LED_current_limit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_15">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>電阻功率消耗(1/4 w 為主)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_18">
     <item row="1" column="0">
      <widget class="QLineEdit" name="W_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_30">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>功耗</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>限流電阻R1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_15">
     <item row="1" column="0">
      <widget class="QLineEdit" name="limit_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_25">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電阻</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_26">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>記號</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="limit_Input_comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>301</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Scientific_computing.qrc">:/LED.bmp</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>D1導通電壓</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_16">
     <item row="1" column="0">
      <widget class="QLineEdit" name="VD_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_27">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電壓</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_16">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>並聯數量(預設1,沒有並聯)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_19">
     <item row="1" column="0">
      <widget class="QLineEdit" name="Parallel_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_31">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>並聯</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>VCC-IO電壓</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_14">
     <item row="1" column="0">
      <widget class="QLineEdit" name="VCCIO_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_24">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電壓</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>D1電流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="1" column="0">
      <widget class="QLineEdit" name="D1_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_21">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電流</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_23">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>記號</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="D1_input_comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_17">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>串聯數量(預設1,沒有串聯)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_20">
     <item row="1" column="0">
      <widget class="QLineEdit" name="Series_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_32">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>串聯</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Scientific_computing.qrc"/>
 </resources>
 <connections/>
</ui>
